config/registry/aws/modules/lambda-function.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "aws-lambda"
+linkTitle: "aws-lambda"
+date: 2021-12-02
+draft: false
+weight: 1
+description: Creates a lambda deployment via opta.
+---
+
+*NOTE* The lambda module is currently in beta, with only the following functionality. Enhanced features and integration 
+with other opta modules will be added in subsequent releases (and expedited as per user need). Please 
+[reach out to the opta team](https://slack.opta.dev/) for any questions/requests.
+
+Creates an [AWS Lambda](https://docs.aws.amazon.com/lambda/index.html) deployment via opta from the given zip file plus
+much more setup. Currently, it also handles:
+
+* Setting up a cloudwatch log group for your function logs
+* Setting up an iam role for your lambda through which to grant permissions to.
+* Exposing the lambda to a public uri if you so wish
+* Allowing the user to pass in IAM policies to give to the lambda
+* Setting up security group and network location if you have the `aws-base` module in your environment.
+
+tl;dr -- this will deploy a hello world lambda
+```yaml
+name: testing-lambda
+org_name: myorg
+providers:
+  aws:
+    region: us-east-1
+    account_id: XXXXXXXXX
+modules:
+  - type: lambda-function
+    expose_via_domain: true
+```
+
+## Adding your Own Function
+Opta supports the [zip file form](https://docs.aws.amazon.com/lambda/latest/dg/configuration-function-zip.html) of
+lambda deployments. For this deployment, simply zip up your code into a zip file and pass its location in via the
+`filename` input. You should also specify the [runtime](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html)
+for your lambda (what language+version you are using). Supposing you created a zip file called baloney.zip, your opta.yml
+should look like the following:
+
+```yaml
+name: testing-lambda
+org_name: myorg
+providers:
+  aws:
+    region: us-east-1
+    account_id: XXXXXXXXX
+modules:
+  - type: lambda-function
+    expose_via_domain: true
+    filename: baloney.zip
+    runtime: "nodejs14.x"
+```
+
+*NOTE* DO NOT add a subdirectory in the zip file-- make its structure flat unless you know what you're doing (this is
+a common mistake with lambda)
+
+## Accessing your function logs
+As is the standard, all of your function invocation logs are stored in cloudwatch under a new log group. You can
+find the log group name and even a helper shortcut url to them in the `opta output`.
+
+## Expose via Public Domain
+You can have your lambda be automatically exposed to the world via a public ui by setting the `expose_via_domain` field
```

Comment:
Can you add a line here like "you can get the default domain by running opta output"?